cpp/src/arrow/filesystem/azurefs.cc

```diff
@@ -119,6 +119,14 @@ Status AzureOptions::ConfigureDefaultCredential(const std::string& account_name)
   return Status::OK();
 }
 
+Status AzureOptions::ConfigureManagedIdentityCredential(const std::string& account_name,
+                                                        std::string const& clientId) {
```

Comment:
Could you use `const XXX` style?

Could you use `snake_case`?

```suggestion
                                                        const std::string& client_id) {
```

Reply:
Sorry, that's another case of not paying enough attention. Its fixed now.